openassessment/templates/openassessmentblock/staff_area/oa_staff_area.html

```diff
@@ -88,10 +88,10 @@ <h2 class="staff-info__title ui-staff__title">
                         </thead>
 
                         <tbody>
-                        {% for item in status_counts %}
+                        {% for key, value in status_counts.items %}
```

Comment:
Old syntax doesn't work with Django>=3.1.0 so it has been replaced with key, value syntax of dictionary items

Reply:
can you drop a link here for Django release notes where this template change was introduced